tests/sphinx/testdata/package/module.py

```diff
@@ -0,0 +1,30 @@
+"""
+Example docstring
+"""
+
+from django.utils.functional import cached_property
+from tests.sphinx.testdata.package.wildcard_module import *  # noqa
+
+from . import other_module
+from .other_module import MyOtherClass
+
+# This file should never get imported. If it is, then something failed already.
+raise Exception
```

Comment:
Did you mean to leave this here? I'm a little lost about the reason to do so.

Reply:
Importing modules is a limitation of `viewcode` and would require a full django setup to work.  This was discussed but it is not an [acceptable solution](https://github.com/django/django/pull/12135#issuecomment-560362400).

The aim of this line is to test that we don't import the module and face the same limitations. 

Do you think there's a way we can do that without just raising an exception here? 

Maybe it is not even needed?

Reply:
Right, I think that we should remove this, since the `raise` in the `__init__.py` file would always get in the way of importing any of the modules anyways. Do you agree?

Reply:
Sure, I can remove it 👍